static/js/services/client-sentinel.js

```diff
@@ -0,0 +1,37 @@
+/*
+  For now this is just one changes listener, but needs to be improved
+  in order to make it more reliable by adding features currently in
+  server-side sentinel such as keeping track of seq numbers.
+ */
+
+angular.module('inboxServices').factory('ClientSentinel',
```

Comment:
I'm not 100% sure we should call this `ClientSentinel` honestly.

I know that's something we want to aspire toward, and I see your comment at the top, but I'm still unsure.

Were it me I would probably just name this `WeathQuintilesWatcher`.

Reply:
Yeah I debated this for a while and left it as `ClientSentinel` in case that work does eventually happen but then someone misses the `WealthQuintilesWatcher` code, but I think that's unlikely to happen so I'll rename it and adjust the comment.